sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -34,6 +34,10 @@
     // TODO temporary until we find final shape of ObjectMapper shimming APIs
     exports com.azure.core.implementation.jackson to com.azure.core.management, com.azure.core.serializer.json.jackson;
 
+    // export core utilities to other core packages.
+    exports com.azure.core.implementation.util to com.azure.http.netty, com.azure.core.http.okhttp,
```

Comment:
Do we need this? What util methods are needed in the HTTP clients? Should they just be moved to public util class?

Reply:
It's so the HttpClient implementations can access the BinaryDataContent types

Reply:
Eventually this is will be needed for core packages that need to interpret BinaryDataContent. The alternative is to make BinaryDataContent public. There's no good reason to open this up outside of azure-core so we take conservative route here.

Reply:
I've previously mentioned I am ok with these conditional exports. It means we retain control and we are not required to treat this as public API.